development/src/components/joinSession_Comp.js

```diff
@@ -17,11 +18,26 @@ function JoinSession() {
     setError("");
   };
 
-  const handleJoinSession = (e) => {
+  const handleJoinSession = async (e) => {
     e.preventDefault();
+
+    const user = {
+      name: "Janet Smith",
+      email: "janetsmith@gmail.com",
+    };
+
+    // Hardcoded session ID
+    const hardcodedSessionId = "76cH5HyEWEhglasWBLl9";
```

Comment:
This will be provided from joinSession modal, you can just pass it in as a parameter